db/migrate/20200217110708_alter_session_sequence_to_cycle.rb

```diff
@@ -1,10 +1,14 @@
 class AlterSessionSequenceToCycle < ActiveRecord::Migration[5.2]
   def up
-    change_column :sessions, :id, :bigint
     if ActiveRecord::Base.connection.adapter_name.downcase == 'postgresql'
-      ActiveRecord::Base.connection.execute(<<-SQL)
-        ALTER SEQUENCE sessions_id_seq MAXVALUE 9223372036854775807 CYCLE;
-      SQL
+      change_column :sessions, :id, :bigint
+      version = Gem::Version.new(ActiveRecord::Base.connection.select_value('SHOW server_version'))
```

Comment:
New problem is that the server version on Debian can't be parsed by `Gem::Version`: `Malformed version number string 11.6 (Debian 11.6-0+deb10u1)`.

From https://ci.centos.org/job/foreman-nightly-debian10-test/108/consoleFull:

```
    [ERROR 2020-02-21T08:09:15 verbose]  /Stage[main]/Foreman::Database/Foreman::Rake[db:migrate]/Exec[foreman-rake-db:migrate]/returns: change from 'notrun' to ['0'] failed: '/usr/sbin/foreman-rake db:migrate' returned 1 instead of one of [0]
    [ WARN 2020-02-21T08:09:36 verbose]  /Stage[main]/Foreman::Database/Foreman::Rake[db:migrate]/Exec[foreman-rake-db:migrate]/returns: rake aborted!
    [ WARN 2020-02-21T08:09:36 verbose]  /Stage[main]/Foreman::Database/Foreman::Rake[db:migrate]/Exec[foreman-rake-db:migrate]/returns: StandardError: An error has occurred, this and all later migrations canceled:
    [ WARN 2020-02-21T08:09:36 verbose]  /Stage[main]/Foreman::Database/Foreman::Rake[db:migrate]/Exec[foreman-rake-db:migrate]/returns:
    [ WARN 2020-02-21T08:09:36 verbose]  /Stage[main]/Foreman::Database/Foreman::Rake[db:migrate]/Exec[foreman-rake-db:migrate]/returns: Malformed version number string 11.6 (Debian 11.6-0+deb10u1)
    [ WARN 2020-02-21T08:09:36 verbose]  /Stage[main]/Foreman::Database/Foreman::Rake[db:migrate]/Exec[foreman-rake-db:migrate]/returns: /usr/share/foreman/db/migrate/20200217110708_alter_session_sequence_to_cycle.rb:5:in `up'
```